### PR TITLE
 Added files for installing kubesphere on minikube issue #2134

### PR DESCRIPTION
**Pull request by enraiha0307**
Added files for steps on installing /uninstalling kubesphere on minikube and its prerequisites, along with the video as well as terminal code demonstration. 
Please review.

### CODE REVIEW

**Comment by FeynmanZhou** [comment]
Hi @enraiha0307 ,

Thanks for your contribution! I think it would be better to add this guide to [this folder](https://github.com/kubesphere/website/tree/master/content/en/docs/installing-on-kubernetes/on-prem-kubernetes). How do you think of it?

/cc @kubesphere/sig-docs

**Comment by enraiha0307** [comment]
> Hi @enraiha0307 ,
> 
> Thanks for your contribution! I think it would be better to add this guide to [this folder](https://github.com/kubesphere/website/tree/master/content/en/docs/installing-on-kubernetes/on-prem-kubernetes). It could be found from the [on-premise category](https://kubesphere.io/docs/installing-on-kubernetes/on-prem-kubernetes/install-ks-on-linux-airgapped/) since Minikube is local K8s distribution that could be regarded as an on-premises solution. What do you think of it?
> 
> /cc @kubesphere/sig-docs
I think you are correct, I'll add it in [on-prem-kubernetes](https://github.com/kubesphere/website/tree/master/content/en/docs/installing-on-kubernetes/on-prem-kubernetes) folder

**Comment by enraiha0307** [comment]
Hi @FeynmanZhou, I have added the guide in [on-prem-kubernetes](https://github.com/kubesphere/website/tree/master/content/en/docs/installing-on-kubernetes/on-prem-kubernetes) folder.
Tell me if any more changes are needed.

**Comment by FeynmanZhou** [comment]
Hi @enraiha0307,

Sorry for the late reply. I have reviewed this guide and your video, it looks quite professional! You can preview it [here](https://deploy-preview-2412--kubesphere-v3-1.netlify.app/docs/installing-on-kubernetes/on-prem-kubernetes/installing-kubesphere-on-minikube/). I request to do the following updates: 

1. Do not remove this [build.yml](https://github.com/kubesphere/website/pull/2412/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721), it is used to create the CI/CD workflow.
2. Please remove these three files since they are duplicated with other places. Just preserve one installation guide in the `on-premises` folder.
image

**Comment by enraiha0307** [comment]
> Hi @enraiha0307,
> 
> Sorry for the late reply. I have reviewed this guide and your video, it looks quite professional! You can preview it [here](https://deploy-preview-2412--kubesphere-v3-1.netlify.app/docs/installing-on-kubernetes/on-prem-kubernetes/installing-kubesphere-on-minikube/). I request to do the following updates:
> 
>     1. Do not remove this [build.yml](https://github.com/kubesphere/website/pull/2412/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721), it is used to create the CI/CD workflow.
> 
>     2. Please remove these three files since they are duplicated with other places. Just preserve one installation guide in the `on-premises` folder.
>        image

Hi @FeynmanZhou  I have preserved the guide to one page.
And about the [build.yml](https://github.com/kubesphere/website/pull/2412/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721), I am not deleting it. It doesn't come when I rebase my branch on master. It only shows in the master branch.

**Comment by rayzhou2017** [comment]
@Patrick-LuoYu @harrisonliu5 Please help review this PR.

File: content/en/docs/installing-on-kubernetes/on-prem-kubernetes/installing-kubesphere-on-minikube.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+linkTitle: "Installing KubeSphere on Minikube"
+weight: 2200,
+showSubscribe: true
+---
+
+In addition to installing KubeSphere on a Linux machine, you can also deploy it on minikube cluster. This tutorial demonstrates the general steps of completing a minimal KubeSphere installation on Minikube.
+
+## Prerequisites
+
+- To install KubeSphere 3.2.1 on Minikube, your Minikube version must be v1.23.x, v1.24.x, v1.25.x.
```
Comment (Patrick-LuoYu) [inline content/en/docs/installing-on-kubernetes/on-prem-kubernetes/installing-kubesphere-on-minikube.md:14]: KubeSphere v3.3.0 is available now. If this document applies to KubeSphere v3.3.0, we should change "KubeSphere 3.2.1" to "KubeSphere v3.3.0".

BTW, the master branch of this repository contains documents of KubeSphere v3.3.0. If this document applies only to KubeSphere v3.2.1, it should be sent to the release-3.2.1 branch of this repository.

Comment (enraiha0307) [reply content/en/docs/installing-on-kubernetes/on-prem-kubernetes/installing-kubesphere-on-minikube.md:14]: Okay, thanks for the review. This document is meant for Kubesphere'ss latest version. I'll update it after testing it on v3.3.0.

Comment (enraiha0307) [reply content/en/docs/installing-on-kubernetes/on-prem-kubernetes/installing-kubesphere-on-minikube.md:14]: Hi, I realized this document contains a video demonstration for Kubesphere 3.2.1, So, should I open up a new pull request for sending this document to release-3.2 branch? 
Or is it possible to change it from github only(like I see there is an option to change the base but it gives the following warning "Some commits from the old base branch may be removed from the timeline, and old review comments may become outdated." ) 
Which way would be the safest?

Comment (Patrick-LuoYu) [reply content/en/docs/installing-on-kubernetes/on-prem-kubernetes/installing-kubesphere-on-minikube.md:14]: On your local machine, you can create a branch synchronized with release-3.2, cherry-pick your commit to the branch, and then create a new pull request.

Comment (enraiha0307) [reply content/en/docs/installing-on-kubernetes/on-prem-kubernetes/installing-kubesphere-on-minikube.md:14]: I have opened up a new pull request. https://github.com/kubesphere/website/pull/2554

**Comment by Bettygogo2021** [comment]
/lgtm
/assign @patrickluoyu
Pls help me merge this PR.

**Pull request merged**